app/authentication/authentication_token_manager.rb

```diff
@@ -0,0 +1,15 @@
+# app/authentication/authentication_token_manager.rb
+class AuthenticationTokenManager
+  class << self
+    def encode(payload)
+      JWT.encode(payload, Rails.application.secrets.secret_key_base)
```

Comment:
I think you need to explicitly set your encoding method here, did you tried the endpoints? As far as i remember this should break when executed if it has no encryption method

Reply:
I tried `api/v1/users/sessions` with a valid user password combination and it successfully returned the access token and the renew_id. Also tested with a non valid user and show the error correctly. Is there anything else I need to try?